src/GridFS/Bucket.php

```diff
@@ -821,7 +821,7 @@ private function resolveStreamContext(string $path, string $mode, array $context
                 'filename' => $filename,
                 'options' => $context + [
                     'chunkSizeBytes' => $this->chunkSizeBytes,
-                    'disableMD5' => $this->disableMD5,
+                    'disableMD5' => true,
```

Comment:
I don't think we need to go this far, and create a divergence between this stream alias API and the Bucket itself. Why not just create a ticket to change the default value of `Bucket::$disableMD5` and target the same release?

Or would that be considered a BC break even though the options API itself remains unchanged?

Reply:
Reverted the default to the bucket config.